src/shogun/mathematics/linalg/LinalgBackendEigen.h

```diff
@@ -55,6 +55,8 @@ namespace shogun
 	    Type beta, Container<Type>& result) const;
 		DEFINE_FOR_NUMERIC_PTYPE(BACKEND_GENERIC_IN_PLACE_ADD, SGVector)
 		DEFINE_FOR_NUMERIC_PTYPE(BACKEND_GENERIC_IN_PLACE_ADD, SGMatrix)
+		BACKEND_GENERIC_IN_PLACE_ADD(complex128_t, SGVector);
+		BACKEND_GENERIC_IN_PLACE_ADD(complex128_t, SGMatrix);
```

Comment:
The issue is that you only added this to the declaration. You have to add this to the implementation too, which is in `BasicOps.cpp`

Reply:
And then let's see if Eigen let's you add complex numbers...